packages/salesforcedx-vscode-lightning/test/deprecatedClassName.test.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import * as path from 'path';
+import * as vscode from 'vscode';
+
+describe('SLDS Deprecated Class Name', () => {
+  let res: vscode.Uri[];
+
+  before(async () => {
+    if (vscode.workspace.rootPath) {
+      res = await vscode.workspace.findFiles(
+        path.join('**', 'DemoComponent.cmp')
+      );
+      await vscode.workspace
```

Comment:
@engai Since you open the file, you should probably have an after that closes it too (just to be tidy). So far it's not causing problems but as we have more tests in the future, I think it's good for each test to clean up after itself.

Reply:
Added an after to close the editor. Looks like closeFile is deprecated.